utilities/transfer-cards/main.go

```diff
@@ -43,7 +48,87 @@ func getProject(client *github.Client, ctx context.Context, projectName string)
 		}
 		opt.Page = resp.NextPage
 	}
-	return nil, fmt.Errorf("project '%s' not found in repo %s/%s", projectName, *repoOwner, *repoName)
+
+	// don't want to create a project if the project is a source project
+	if source {
+		return nil, fmt.Errorf("project '%s' not found in repo %s/%s", projectName, *repoOwner, *repoName)
+	}
+	// if this is a dry run do not create the project
+	if *dryrun {
+		return nil, fmt.Errorf("project '%s' not found in repo %s/%s if you would like the utility to create the project rerun without the dryrun option\n", projectName, *repoOwner, *repoName)
+	}
+
+	// if project was not found try creating the project
+	if getProjectErr != nil || project == nil {
```

Comment:
Is there ever any scenario where we'd get to this point and `project` wouldn't be `nil`?